app/models/host.rb

```diff
@@ -145,6 +145,10 @@ class Host < ApplicationRecord
   virtual_total :v_total_vms, :vms
   virtual_total :v_total_miq_templates, :miq_templates
 
+  has_many :host_hardwares, :class_name => 'Hardware', :source => :hardware, :dependent => :nullify
```

Comment:
think `alias_attribute :hardware, :host_hardwares` would work here. (not sure the order of those args)
also, can you move these up with other has many?

Reply:
this is totally my bad. I asked you to define it this way.
I promise not to flip flop back if it is any consolation